lib/game.rb

```diff
@@ -59,4 +60,20 @@ def play
   def current_player
     turn_count(@board.cells).even? ? @player1 : @player2
   end
+
+  def select_player_order
+    puts "\n Choose 1 to play first, or 2 to play second: "
+    input = gets.chomp
```

Comment:
the getting of input could be extracted out

Reply:
commit 96154987fdacd649f0b93184b45880112872dfea